src/mesh/RadioInterface.cpp

```diff
@@ -36,10 +36,18 @@ const RegionInfo regions[] = {
 
         audio_permitted = false per regulation
      */
-    RDEF(EU868, 869.4f, 869.65f, 10, 0, 16, false, false),
+    RDEF(EU868, 869.4f, 869.65f, 10, 0, 27, false, false),
 
     /*
         https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf
+
```

Comment:
This comment is added to the section for the CN region. Please update the comment for the EU868 region.

Will take a look at this closer soon. Thanks!

Reply:
Sending a fix